dependencies/recommended.txt

```diff
@@ -21,3 +21,4 @@ matplotlib
 git+https://github.com/microsoft/nn-Meter.git#egg=nn_meter
 sympy
 timm >= 0.5.4
+datasets == 2.10.1
```

Comment:
Any particular reason to freeze the version?

Reply:
good suggestion, there is no need to freeze the version. I will modify it.